packages/tools/u-boot/package.mk

```diff
@@ -142,6 +142,18 @@ makeinstall_target() {
         cp -PRv $DISTRO_DIR/$DISTRO/splash/boot-logo.bmp.gz $INSTALL/usr/share/bootloader
       fi
       ;;
+  esac
+
+  case $PROJECT in
```

Comment:
Do yo know a simpler way to manage this ?

I thought about a case "$DEVICE$PROJECT" and "Odroid_C2*" would match in either case

Reply:
Simpler or easier to follow maybe... case in a case?  Really going to be personal opinion.
`case $DEVICE in`
`Odroid_C2) blah blah ;;`
`*) case $PROJECT in`
`Odroid_C2) blah blah ;;`
`esac`
`esac`